src/snowcli/utils.py

```diff
@@ -604,3 +611,14 @@ def sql_to_python_return_type_mapper(resource_return_type: str) -> str:
     }
 
     return mapping.get(resource_return_type.lower(), resource_return_type.lower())
+
+
+def path_resolver(path_to_file: str):
+    if platform.system() == "Windows" and "~1" in path_to_file:
```

Comment:
Why you use 2 different conditions? `platform.system() == "Windows"` and `sys.platform == "win32"`

Reply:
Reformatted to single check